minisite.module

```diff
@@ -2,26 +2,15 @@
 
 /**
  * @file
- * Main functions of the module.
+ * Main functions of the Minisite module.
  */
 
 use Drupal\Core\Routing\RouteMatchInterface;
+use Drupal\file\FileInterface;
+use Drupal\minisite\Exception\ArchiveException;
+use Drupal\minisite\Minisite;
 
-define('MINISITE_UPLOAD_PATH', 'minisite/upload');
-define('MINISITE_ASSET_PATH', 'minisite/static');
-define('MINISITE_EXTENSIONS_WHITELIST', 'html htm js css png jpg gif svg pdf doc docx ppt pptx xls xlsx tif xml txt woff woff2 ttf eot');
-define('MINISITE_EXTENSIONS_BLACKLIST', 'exe scr bmp');
-define('MINISITE_EXTENSIONS_NOREWRITE', 'pdf doc docx ppt pptx xls xlsx tif xml txt');
-define('MINISITE_MAXFILESIZE', format_size(file_upload_max_size()));
-
-// Load lib for Minisite.
-require_once dirname(__FILE__) . '/includes/minisite.loader.inc';
```

Comment:
all of these, except for `minisite.theme.in`, were used (partially) for aliases, so they are removed now.